app/data/archivePaths.ts

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+const COVERAGE = 'Step 1 - Eligibility Mapping & Statistics';
+const TEMPLATE_UPLOADS = 'Step 2 - Templates';
+const SUPPORTING_DOCUMENTS = 'Step 4 - Supporting Documents';
+
+const archivePaths = {
```

Comment:
There's a risk here for future maintenance, which is that two pages might have fields that have the same name.

Rather than `archivePaths` being an object, you may want to consider making it an array, containing the form path for each element.
Additionally, you could use a simple string and use a `replace` to insert the ccbcId and fileName, rather than copying the same function signature for every template. 

It would look like this, which I think would make this file more concise and easier to read:

```ts
const archivePaths = [
  { formPath: 'templateUploads.eligibilityAndImpactsCalculator', archivePath: `/$id/${TEMPLATE_UPLOADS}/$id - Template 1 - $filename`},
  { formPath: 'templateUploads.detailedBudget', archivePath: `/$id/${TEMPLATE_UPLOADS}/$id - Template2 - $filename`},
]
``` 
You would need a little bit of documentation at the top indicating which variables are supported (only `$id` and `$filename` for now?).

You can use [`lodash.get`](https://www.npmjs.com/package/lodash.get) to easily retrieve the formData for any `formPath`

Reply:
Additionally, the first `/$id` could be omitted given that it's on every file

Reply:
I wasn't aware this was an intentional choice. If the application behaviour relies on this constraint on the form schema, it should be documented somewhere and potentially enforced with a static or runtime check. I'd suggest creating a technical debt card to document this restriction and the reasons behind it.